Rego/EXOConfig.rego

```diff
@@ -496,10 +507,12 @@ tests[{
     "Criticality" : "Should",
     "Commandlet" : ["Get-HostedConnectionFilterPolicy"],
     "ActualValue" : input.conn_filter,
-    "ReportDetails" : ReportDetailsBoolean(Status),
+    "ReportDetails" : ReportDetailsString(Status, ErrorMessage),
     "RequirementMet" : Status
 }]{
-    Status := count(ConnFiltersWithSafeList) == 0
+    ConnFilterPolicies := ConnFiltersWithSafeList
```

Comment:
Shouldn't `ConnFilterPolicies` be the ActualValue in line 509 instead of `input.conn_filter`?